tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -51,13 +51,7 @@
 # @result namespacestr [String] Returns 'sudo ip netns exec vrf <cmd>'
 # command string for 'cisco' platform.
 def get_namespace_cmd(host, cmdstr, options)
-  case host['platform']
-  when /cisco/
-    agentvrf = options[:HOSTS][host.to_s.to_sym]['vrf']
-    return "sudo ip netns exec #{agentvrf} " + cmdstr
-  else
-    return cmdstr
-  end
+  return cmdstr
```

Comment:
Ah, okay. Can you point me at the beaker code where these vrf changes are being made? Thanks.